src/gui/gui2_advancedscrolltext.h

```diff
@@ -29,8 +36,8 @@ class GuiAdvancedScrollText : public GuiElement
     GuiAdvancedScrollText* setTextSize(float text_size) { this->text_size = text_size; return this; }
 
     unsigned int getEntryCount() const;
-    string getEntryText(int index) const;
```

Comment:
I think most of the changes in this pull request are good. But I don't like taking unsigned ints here. I sort of understand the reasoning and that it most likely won't cause any problems. It's inconsistent with the rest of the code which pretty much never uses unsigned ints.

Reply:
I'll eliminate the usage for getEntryCount() too, then, since that was likely part of the motivation for aligning the others (they do a casted comparison to getEntryCount())

Reply:
Neverminded, I found many other cases where it's compared to other vector::size()'s or unsigned local iterators. Are you SURE you don't like unsigned ints? 😆

Reply:
It's fine for local iterators. But I don't think they are used anywhere in API, and sometimes in casts when comparing with vector sizes. But in general, I try to keep numbers unsigned, so you don't have issues with checks like `if (n - 3 < x)`